### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bennigraf**
Hi,

I'd like to propose to document how to specify the request range when creating a read stream. I had to dig into the source code to find out how to do this; I thought it might help others to have this available in the readme.

What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,16 @@ client
...
 ```
 
+If you want to request only partial data, you can specify the range in the options argument of the method:
+
+```javascript
+client
+    .createReadStream(
+        "/video.mp4", 
+        { range: { start: start, end: end } }
+    ).pipe(fs.createWriteStream("~/video.np4"));
```
Comment (perry-mitchell) [inline README.md:157]: Should be `.mp4`

File: README.md [hunk 2]
```diff
@@ -147,6 +147,16 @@ client
     .pipe(fs.createWriteStream("~/video.np4"));
 ```
 
+If you want to request only partial data, you can specify the range in the options argument of the method:
```
Comment (perry-mitchell) [inline README.md:150]: Could you change this to: "If you want to stream only part of the file, you can specify the `range` in the options argument:"

**Review by perry-mitchell — CHANGES_REQUESTED** [review]

**Comment by perry-mitchell** [comment]
@bennigraf Great, thanks! Just a minor change :)

File: README.md (hunk 1 above)
Comment (bennigraf) [reply README.md]: Sure!

File: README.md (hunk 2 above)
Comment (bennigraf) [reply README.md]: Yep, sounds better

**Comment by bennigraf** [comment]
Thanks for your review, I updated the PR according to your requests. Thanks for _your_ work on `webdav-client` :-)

**Review by perry-mitchell — APPROVED** [review]

**Pull request merged**